plugins/by-name/otter/default.nix

```diff
@@ -19,7 +19,7 @@ helpers.neovim-plugin.mkNeovimPlugin {
         "otter"
         "addCmpSources"
       ]
-      "Otter is now supported by `plugins.cmp.autoEnableSources`, adding `otter` to `cmp` sources will enable this plugin."
+      "The `plugins.otter.addCmpSources` option is deprecated and will be rmoved. A quote from the README: \"If you previously used the otter nvim-cmp source, you can remove it, as the completion results now come directly via the cmp-nvim-lsp source together with other language servers.\""
```

Comment:
It's inaccurate to say "will be removed". If we have a removal-module then the option _is_ already removed.

Also, note that the removal module itself already includes text to this effect:

https://github.com/NixOS/nixpkgs/blob/c976169e5c334ae42bdf239a868e6a218df63e67/lib/modules.nix#L1112-L1115

```
The option definition `${showOption optionName}' in ${showFiles opt.files} no longer has any effect; please remove it.
${replacementInstructions}
```

Reply:
I removed the `will be removed` words.

Reply:
I am not really sure how I can improve it more.